Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -261,6 +275,12 @@ bool WebXROpaqueFramebuffer::setupFramebuffer()
     }
 
     if (m_attributes.antialias && m_context.isWebGL2()) {
+        if (m_resolvedFBO)
+            gl.deleteFramebuffer(m_resolvedFBO);
```

Comment:
If only there was some kind of automatic GL object helper so we couldn't forget to do this.